kahuna/public/js/leases/leases.html

```diff
@@ -31,10 +31,24 @@
 
   <select id="access-select" ng:model="ctrl.access" ng-required="required">
       <option ng-selected="true" value="">Please select access</option>
-      <option value="allow-use">Allow cropping</option>
-      <option value="deny-use">Deny cropping</option>
+      <optgroup label="Cropping">
+          <option value="allow-use">Allow cropping</option>
+          <option value="deny-use">Deny cropping</option>
+      </optgroup>
+      <optgroup label="Syndication">
+          <option value="allow-syndication">Allow syndication</option>
+          <option value="deny-syndication">Deny syndication</option>
+      </optgroup>
   </select>
+
+  <div ng:if="ctrl.access === 'deny-syndication'">
+    <input type="checkbox"
+           ng:model="ctrl.showCalendar" />
+    Expires
+  </div>
+
   <gu-date-range class="lease__date"
+     ng:if="ctrl.access !== 'allow-syndication' && !(ctrl.access === 'deny-syndication' && ctrl.showCalendar === false)"
```

Comment:
minor: its better to push this down to the controller to keep the template simple

e.g.
```js
ctrl.foo = ctrl.access !== 'allow-syndication' && !(ctrl.access === 'deny-syndication' && ctrl.showCalendar === false)
```

```html
<gu-date-range ng:if="ctrl.foo">
```